docroot/modules/custom/cs_generated_content/generated_content/node/civictheme_page_variations/17.banner_component.content.inc

```diff
@@ -1,126 +0,0 @@
-<?php
```

Comment:
Deleted all the individual banner component generation as these components are tested in their component generated content.